.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
```

Comment:
wow thanks ;)
it is worth to have a monthly bot for major versions also

Reply:
Just added :smile:

Reply:
I didn't find a way of enabling major montlhy. 

So, i keep major/minor/patch weekly

Reply:
Below is the way i do it Ignoring majors from others and set a separate for major
```
version: 2

updates:
  - package-ecosystem: "npm"
    directory: "/"
    target-branch: "dependencies"
    groups:
      prod-dependencies:
        dependency-type: "production"
        update-types:
          - "minor"
          - "patch"
      dev-dependencies:
        dependency-type: "development"
        update-types:
          - "minor"
          - "patch"
    ignore:
      - dependency-name: "npm"
        update-types: [ "version-update:semver-major" ]
    reviewers:
      -  ....
    schedule:
      interval: "weekly"
      day: "monday"
      time: "11:00"
      timezone: "Europe/Paris"

  - package-ecosystem: "npm"
    directory: "/"
    target-branch: "major-dependencies"
    groups:
      prod-dependencies:
        dependency-type: "production"
        update-types:
          - "major"
      dev-dependencies:
        dependency-type: "development"
        update-types:
          - "major"
    ignore:
      - dependency-name: "npm"
        update-types: [ 
          "version-update:semver-minor",
          "version-update:semver-patch"
        ]
    reviewers:
      - "......"
    labels:
      - 'major-dependency'
    schedule:
      interval: "monthly"

```

Reply:
Hummmmm You use differents target-branches :thinking: 

I'm curious here, 
1. How you keep `major-dependencies` and `dependencies` in sync with `main`? Is it automatic? 
2. Dependabot opens PR against `major-dependencies` and `dependencies`, after the merge it needs open a PR to `main`? Could be automatic?

As you have more knowledge about `dependabot`, open up a PR to my branch :pray:

Reply:
This is a detail 
I autoerge for minor and patch after tests sucess but for major I merge manually after testing by devs 

This is a criticity concern